### PR TITLE
Upgrade all elasticsearch versions to latest versions

### PR DESCRIPTION
**Pull request by andytson-inviqa**
Even if a platform doesn't explicitly say it supports beyond 7.10, it will need upgrading

Spryker's elasticsearch 6 can use 6.8.22

### CODE REVIEW

File: src/akeneo/harness.yml [hunk 1]
```diff
@@ -134,11 +134,6 @@ attributes:
     image:
       console: = 'my127/akeneo:' ~ @('php.version') ~ '-fpm-buster-console'
       php-fpm: = 'my127/akeneo:' ~ @('php.version') ~ '-fpm-buster'
-  elasticsearch:
-    image: elasticsearch
-    tag: '7.10.1'
-  mysql:
-    tag: '8.0'
```
Comment (kierenevans) [inline src/akeneo/harness.yml:141]: huh?

Comment (andytson-inviqa) [reply src/akeneo/harness.yml:141]: it's 8.0 in base

Comment (kierenevans) [reply src/akeneo/harness.yml:141]: Ah is the default 8.0 now?

**Review by kierenevans — DISMISSED** [review]

File: src/akeneo/harness.yml (hunk 1 above)
Comment (andytson-inviqa) [reply src/akeneo/harness.yml:141]: yep, so devs don't suffer the performance penalty of mysql/mysql-server:5.7 also not supporting arm64 (though it handles the platform choice better than mysql:5.7 insisting on platform being hard-coded

**Comment by andytson-inviqa** [comment]
Update elasticsearch to 7.16.2 for additional CVE fix

**Comment by andytson-inviqa** [comment]
Exclude upgrading Spryker elasticsearch

it will be done in #655

**Review by coledave — APPROVED** [review]

**Pull request merged**